### PR TITLE
Update README to streamline core focus of the FAST project

### PR DESCRIPTION
**Pull request by janechu**
# Pull Request

## 📖 Description

This pull request simplifies the root level README in accordance with a re-alignment on core FAST goals.

We will be re-focusing the project around the `@microsoft/fast-element` package and a number of existing packages will have RFCs in the future for discussion around deprecation strategy.

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,46 +40,21 @@ For an in-depth explanation of FAST [see our docs introduction](https://www.fast
...
-
-The `@microsoft/fast-router` package contains a history-based navigation and routing solution designed around Web Components. By using `fast-router`, you can create multi-page and full application experiences. The router works with any Web Components, but has special support for Web Components built on FAST.
-
 ### `@fluentui/web-components`
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 [![npm version](https://badge.fury.io/js/%40fluentui%2Fweb-components.svg)](https://badge.fury.io/js/%40fluentui%2Fweb-components)
 
-`@fluentui/web-components` is a library of Web Components that *composes* `@microsoft/fast-foundation`. `@fluentui/web-components` makes use of different stylesheets and may include components that specifically support Microsoft's Fluent design language.
+`@fluentui/web-components` is a library of Web Components based on the Fluent design language.
```
Comment (scomea) [inline README.md:48]: Shouldn't "fluent is based on FAST" be mentioned here?

Comment (janechu) [reply README.md]: Good point, added.

File: README.md [hunk 2]
```diff
@@ -19,21 +18,15 @@
...
 [![Twitter](https://img.shields.io/twitter/follow/fast_ui.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=fast_ui)
 
-This is the FAST monorepo, containing web component packages, tools, examples, and documentation. FAST tech can be used à la carte or as a suite to build enterprise-grade websites, applications, components, design systems, and more.
-
 :star: We appreciate your star, it helps!
 
 ## Introduction
 
-FAST is a collection of technologies built on Web Components and modern Web Standards, designed to help you efficiently tackle some of the most common challenges in website and application design and development.
+FAST is dedicated to providing support for native Web Components and modern Web Standards, designed to help you efficiently tackle some of the most common challenges in website and application design and development.
```
Comment (radium-v) [inline README.md:25]:
```suggestion
FAST is dedicated to providing support for native Web Components and modern Web Standards, and it's designed to help you efficiently tackle some of the most common challenges in website and application design and development.
```

File: README.md [hunk 3]
```diff
@@ -47,46 +40,21 @@ For an in-depth explanation of FAST [see our docs introduction](https://www.fast
...
-
-The `@microsoft/fast-router` package contains a history-based navigation and routing solution designed around Web Components. By using `fast-router`, you can create multi-page and full application experiences. The router works with any Web Components, but has special support for Web Components built on FAST.
-
 ### `@fluentui/web-components`
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 [![npm version](https://badge.fury.io/js/%40fluentui%2Fweb-components.svg)](https://badge.fury.io/js/%40fluentui%2Fweb-components)
 
-`@fluentui/web-components` is a library of Web Components that *composes* `@microsoft/fast-foundation`. `@fluentui/web-components` makes use of different stylesheets and may include components that specifically support Microsoft's Fluent design language.
+`@fluentui/web-components` is a library of Web Components based on the Fluent design language and `@microsoft/fast-element`.
```
Comment (radium-v) [inline README.md:48]:
```suggestion
`@fluentui/web-components` is a library of Web Components based on the Fluent design language, built with `@microsoft/fast-element`.
```

**Review by chrisdholt — APPROVED** [review]

**Review by radium-v — APPROVED** [review]

**Pull request merged**